src/app/components/kudos-challenge-completed/kudos-challenge-completed.component.js

```diff
@@ -0,0 +1,90 @@
+(function () {
+
+    var CompletedChallengesController = function ($scope, $httpParamSerializer, CompletedChallengesService, Challenges, Resources) {
+
+        $scope.completedChallengesCollection = [];
+        $scope.showList = false;
+
+        $scope.checkList = checkList;
+        $scope.acornPlural = acornPlural;
+        $scope.getWinner = getWinner;
+        $scope.userEmail = Resources.getCurrentUserEmail();
+        $scope.showMore = showMore;
+        $scope.showLess = showLess;
+        $scope.showMoreButton = showMoreButton;
+        var showMoreLimit = 5;
+        $scope.completedChallengesLimit = 5;
+
+        CompletedChallengesService.completedChallenges().then(function (val) {
```

Comment:
Please look at my comment in this factory. Probably could use different naming and a different service altogether.